plugin/trino-tpch/src/main/java/io/trino/plugin/tpch/DoubleTypeMapping.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.tpch;
+
+import io.trino.spi.type.DecimalType;
+import io.trino.spi.type.DoubleType;
+import io.trino.spi.type.Type;
+
+import static java.util.Objects.requireNonNull;
+
+public enum DoubleTypeMapping
+{
+    DOUBLE(DoubleType.DOUBLE),
+    DECIMAL(DecimalType.createDecimalType(12, 2));
```

Comment:
Standard says:
>  Decimal means that the column must be able to represent values in the range -9,999,999,999.99 to
+9,999,999,999.99 in increments of 0.01; the values can be either represented exactly or interpreted to be in
this range;

So that matches decimal 12,2. The question is why do we want to use double (other than legacy reasons)?

FYI: http://www.tpc.org/tpc_documents_current_versions/pdf/tpc-h_v2.17.1.pdf

Reply:
> The question is why do we want to use double (other than legacy reasons)?

would switching the type here "invalidate" benchmark results?
cc @sopel39

Reply:
I don't think anyone is using TPCH connector for benchmarking. I guess it could be used to generate data, but actual benchmarks should use production like connectors like `hadoop-hive2`.

Reply:
Are there any other columns with `DOUBLE` type? It's used in some tests where a DOUBLE column was needed.

Reply:
My feature is off by default.  I added this so I could generate data in with decimal for other purposes.

Reply:
> I don't think anyone is using TPCH connector for benchmarking. 

@kokosing of course, not directly

> I guess it could be used to generate data,

and hence schema (types) matters